flaml/nlp/huggingface/training_args.py

```diff
@@ -1,9 +1,7 @@
 import argparse
 from dataclasses import dataclass, field
 
-from ...data import (
-    NLG_TASKS,
-)
+from flaml.automl.task import NLG_TASKS
```

Comment:
The `...data` is meant to be kept as the relative path to be consistent with the other files. Is there any reasons to change it to the absolute path?